app-tasks/dags/ingest_project_scenes.py

```diff
@@ -41,7 +42,8 @@
 batch_job_definition = os.getenv('BATCH_INGEST_JOB_NAME')
 batch_job_queue = os.getenv('BATCH_INGEST_JOB_QUEUE')
 hosted_zone_id = os.getenv('HOSTED_ZONE_ID')
-jar_path = os.getenv('BATCH_JAR_PATH', 'rf-batch-53937da.jar')
+jar_path = 'rf-batch-ingest-status.jar'
```

Comment:
this should be removed once it's time to merge, but for now it points to the jar with the changes